proyecto/src/test/java/com/proyecto/proyecto/CategoriasTest.java

```diff
@@ -0,0 +1,89 @@
+package com.proyecto.proyecto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CategoriasTest {
+
+	@Autowired
+	RepositorioCategorias repoCategorias;
+
+	@Test
+	void testCrearCategoria() {
+		Categoria categoria = new Categoria();
+		categoria.setId(10);
+		categoria.setNombre("Construccion");
+		categoria.setDescripcion("Trabajos relacionados con la construccion");
+		repoCategorias.save(categoria);
+		assertNotNull(repoCategorias.findById(10).get());
```

Comment:
#should además habría que verificar los otros datos establecidos: nombre y descripción.